Rename reportingCurrency to reportCurrency on experimental Post/Put Organization

wmdmgv opened the pull request:
according to https://github.com/Rebilly/RebillyReportsAPI/pull/86#issuecomment-559332051

wmdmgv commented:
@adamaltman  so, I have removed duplicates https://github.com/Rebilly/RebillyReportsAPI/pull/87/commits/3d6d43f796b968ff75e5663c20a28b64d8da8b13  (sorry, forgot to remove it)

concerning
>  With a PUT request it says I can create an organization and all that is required is a name. With a POST request I need a name and a website
we have a remark `(required for create operation and not used for update)` for website
https://github.com/Rebilly/RebillyReportsAPI/pull/87/files#diff-fc31b3715a481477996aa307f833895dL11
or how do you suggest define as required field for PUT in case of creation but not used in case of edit ?  :innocent:

concerning
 > What about country?
https://github.com/Rebilly/RebillyReportsAPI/pull/84#discussion_r348992410 
we decided to use it as not required because we have a default value, as well as fronted anyway sends a country there

adamaltman commented:
@wmdmgv PUT requires a full resource.  Therefore, it should not be available for creation, and there are possible properties missing.

Regarding country, I think it should be required -- we should use the geo-ip country, which the frontend can supply to us.

adamaltman commented:
Why do we need PUT at all?

wmdmgv commented:
So, I removed PUT and added PATCH. 
Post
![postOrganization](https://user-images.githubusercontent.com/6420056/69817521-d5417b00-120b-11ea-8b9b-7d12ce2a2dbe.png)

Patch
![patchOrganization](https://user-images.githubusercontent.com/6420056/69817522-d5417b00-120b-11ea-94ce-271a78fd6aa5.png)

adamaltman commented:
Seems wrong.  How do you patch "Website"? 

https://github.com/Rebilly/RebillyReportsAPI/pull/87/files#diff-f9e149196cbed458615e8268f3c037edR8

adamaltman commented on spec/components/requestBodies/PatchOrganization.yaml:
```diff
@@ -1,16 +1,10 @@
       properties:
         name:
-          description: The organization name
+          description: The organization unique name
```
```suggestion
          description: The organization name.  It must be globally unique within Rebilly.
```

adamaltman commented on spec/components/requestBodies/PostOrganization.yaml:
```diff
@@ -48,7 +49,7 @@ content:
           type: string
           maxLength: 50
-        reportingCurrency:
+        reportCurrency:
```
It seems a bad idea to have a default value for this since it **cannot** be changed, ever.

wmdmgv replied:
ok, then seems we should to make reportCurrency as required field too and remove default value.

wmdmgv replied:
@adamaltman so, reportCurrency is required for now

adamaltman commented on spec/components/schemas/Organization.yaml:
```diff
@@ -1,16 +1,12 @@
       - $ref: "#/components/schemas/ResourceId"
   name:
-    description: The organization name
+    description: The organization unique name
```
```suggestion
    description: The organization name.  It must be unique globally within Rebilly.
```

VladimirBerdnik commented on spec/components/requestBodies/PatchOrganization.yaml:
```diff
@@ -1,16 +1,10 @@
       properties:
         name:
-          description: The organization name
+          description: The organization name.  It must be globally unique within Rebilly.
```
```suggestion
          description: The organization name. It must be globally unique within Rebilly.
```
Same in few places in this PR with the same sentence
